docs/src/ctevaluation.html.jinja2

```diff
@@ -516,6 +516,25 @@
 
             $('#body').scrollspy({target: '#wizard-bar', offset: 10})
         });
-    </script>
 
+         function revertField(element) { 
+            //alert("revertField element: element.val(): " + element + " : " + $(element).val()
+            //    + " : " + $(element).prev() + " : " + $(element).prev().val() );
+            ele = $(element).prev();
+            if ($(ele).hasClass("update-data")) {
```

Comment:
.prev() already returns a jQuery object so continuously referencing this as $(ele) is useless. The standard naming convention for using jQuery objects is to just prefix the variable name with a $. If this were kept, I'd suggest you rename and use `$ele`. 

The use of .prev() is dangerous here. This code is VERY specific in that it requires that the preceding element is always the element you want to manipulate. If that were not true, we would end up with incorrect behavior or worse, bad data.